analyzer/src/main/kotlin/managers/utils/DependencyGraphBuilder.kt

```diff
@@ -132,17 +132,21 @@ class DependencyGraphBuilder<D>(
      * explicitly, as they are recorded automatically by _addDependency()_. However, if there are scopes without
      * dependencies, this function can be used to include them into the builder result.
      */
-    fun addScope(scopeName: String) {
+    fun addScope(scopeName: String): DependencyGraphBuilder<D> {
```

Comment:
Style nit: We usually (if possible) implement the "fluent" pattern by using `apply {}` in function expressions.

Reply:
I see. So I'd drop this comment then, right?

Reply:
Which comment? Do you mean commit? No, I was not proposing to drop it, but to implement it like

    fun addScope(scopeName: String): DependencyGraphBuilder<D> =
        apply { scopeMapping.putIfAbsent(scopeName, emptyList()) }

Reply:
Sorry, I meant "commit" instead of "comment". Good that I asked...